infrastructure/images/moduletest-reset-jobs/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.21.3
+
+# Install apk dependencies
+RUN apk add --no-cache nodejs npm bash kubectl && npm install -g zx
```

Comment:
You should pin these dependencies, so you don't risk version drift.

Reply:
done